### ISSUE TITLE
[BUG] Custom CSS styles not activating until 'type="text/css"' is added to corresponding link in HTML head

### ISSUE THREAD

**Issue by shimizurei** [Lissy93/dashy#560]
### Environment

Self-Hosted (Docker)

### Version

2.0.4

### Describe the problem

I had finally gotten the CSS to not throw a 404, but for the life of me I could not get my styles to show up.

UNTIL I changed one thing (using devtools):

How the custom CSS loads in: `<link rel="stylesheet" href="/assets/dashycsstweaks.css">`

What I needed to do to activate my CSS: `<link rel="stylesheet" type="text/css" href="/assets/dashycsstweaks.css">`

I was previously upping the CSS specificity to disgusting levels to no avail:

```
div#dashy div.home[style*="background: url("/assets/wetpavement_bg.jpg") 0% 0% / cover"] {
  background-attachment: fixed !important;
}
```

Would it be possible to add the `type="text/css"` part to the generated link when the app is built?

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Yes, absolutely I can add that
Quick question, how are you specifying the stylesheet in your config?

**Comment by shimizurei** [comment]
Docker compose, under volumes: `- /volume1/docker/dashy/assets/dashycsstweaks.css:/app/public/assets/dashycsstweaks.css`

In conf.yml: `externalStyleSheet: ['/assets/dashycsstweaks.css']` (under `appConfig`).

Every time I've tried to use `externalStyleSheet: '/assets/dashycsstweaks.css'`, an error message shows up in the online edit part saying that `externalStyleSheet` should be an array, despite the documentation saying that is an ok way to reference.

**Comment by Lissy93** [comment]
Thanks, that's helpful - I'd totally forgotten ever building that bit :rofl: 
I'll get a fix pushed out as part of #557 :)

**Comment by Lissy93** [comment]
Merged in #557 :)

**Comment by changbowen** [comment]
@Lissy93 
This seems to break Google fonts loaded via `externalStyleSheet`?

![google font issue](https://user-images.githubusercontent.com/15975872/205478297-da10d5e3-8ed7-40f4-a5f5-56c965153c3f.gif)
